doc_test.go

```diff
@@ -0,0 +1,55 @@
+package pack_test
+
+import (
+	"context"
+	"fmt"
+	"math/rand"
+
+	"github.com/buildpacks/pack"
+)
+
+// This example shows the basic usage of the package: Create a client,
+// call a configuration object, call the client's Build function.
+func Example_build() {
```

Comment:
This is interesting... I'm assuming it doesn't get executed along with the rest of the tests. I wonder if it would be better to document this elsewhere.

Reply:
Ya I'm not sure exactly where else to put it. It seems like if you want the example to be picked up by godoc, this is the recommended format.

Reply:
Would this be something that should be moved to buildpacks/docs?

Reply:
After some conversations we came to the conclusion that this standard practice within the go lang community makes the most sense in this scenario.